src/modules/all.mk

```diff
@@ -23,5 +23,7 @@ endif
 
 ifneq "$(MAKECMDGOALS)" "reconfig"
 src/modules/%/configure: src/modules/%/configure.ac
-	@echo WARNING - may need "'make reconfig'" for AUTOCONF $(dir $@)
+	${Q}if [ $< -nt $@ ]; then \
```

Comment:
GNU Make should check this automatically though.  That's what Makefile rules do.

What else is going on here?

I get the WARNING messages sporadically, but not consistently.

Reply:
I get warnings all the time. :(

Reply:
even with a simple "git clone ...; ./configure; make"

e.g:

```
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_yubikey/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_winbind/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_unix/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_unbound/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_sql/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_sql/drivers/rlm_sql_firebird/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_sql/drivers/rlm_sql_freetds/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_sql/drivers/rlm_sql_db2/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_sql/drivers/rlm_sql_oracle/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_sql/drivers/rlm_sql_sqlite/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_sql/drivers/rlm_sql_unixodbc/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_sql/drivers/rlm_sql_postgresql/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_sql/drivers/rlm_sql_mysql/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_sql/drivers/rlm_sql_cassandra/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_sigtran/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_securid/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_rest/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_radutmp/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_python/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_perl/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_pam/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_opendirectory/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_mschap/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_mruby/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_lua/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_krb5/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_idn/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_example/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_couchbase/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_cache/drivers/rlm_cache_memcached/
WARNING - may need 'make reconfig' for AUTOCONF src/modules/rlm_cache/
```

I am running the below make/autoconf.

```
[jpereira@sugarloaf freeradius-server.git]$ autoconf --version | head -1
autoconf (GNU Autoconf) 2.69
[jpereira@sugarloaf freeradius-server.git]$ make --version | head -1
GNU Make 3.81
[jpereira@sugarloaf freeradius-server.git]$
```